code/modules/admin/verbs/getlogs.dm

```diff
@@ -1,41 +1,63 @@
-/*
-	HOW DO I LOG RUNTIMES?
-	Firstly, start dreamdeamon if it isn't already running. Then select "world>Log Session" (or press the F3 key)
-	navigate the popup window to the data/logs/runtime/ folder from where your tgstation .dmb is located.
-	(you may have to make this folder yourself)
-
-	OPTIONAL: 	you can select the little checkbox down the bottom to make dreamdeamon save the log everytime you
-				start a world. Just remember to repeat these steps with a new name when you update to a new revision!
-
-	Save it with the name of the revision your server uses (e.g. r3459.txt).
-	Game Masters will now be able to grant access any runtime logs you have archived this way!
-	This will allow us to gather information on bugs across multiple servers and make maintaining the TG
-	codebase for the entire /TG/station commuity a TONNE easier :3 Thanks for your help!
-*/
-
-
-//This proc allows download of past server logs saved within the data/logs/ folder.
+/// This proc allows download of past server logs saved within the data/logs/ folder.
 /client/proc/getserverlogs()
 	set name = "Get Server Logs"
 	set desc = "View/retrieve logfiles."
 	set category = "Admin"
 
-	var/path = browse_files("data/logs/")
-	if(!path)
+	access_file_by_browsing_path(usr, "data/logs/")
+
+/// This proc allows download of past server logs saved within the data/logs/ folder by specifying a specific round ID.
+/client/proc/get_server_logs_by_round_id()
+	set name = "Get Round Logs"
+	set desc = "View/retrieve logfiles for a given round."
+	set category = "Admin"
```

Comment:
Can you add a `check_rights()` to this, just to be safe?

Reply:
Done.